Overlapping text with menu-items fixed

Usamaahmad751 opened the pull request:
menu-items were overlapping with the text-behind them. So I have added standard backgrounds to menu-items in both and light mode to make them properly visible and better user experience.

Usamaahmad751 committed:
Overlapping text with menu-items fixed

menu-items were overlapping with the text-behind them. So I have added standard backgrounds to menu-items in both and light mode to make them properly visible and better user experience.

9inpachi commented on src/components/Menu/MenuItem/MenuItem.scss:
```diff
@@ -12,9 +13,19 @@
     }
 }
 
+.dark-mode {
```
The css under `.dark-mode` is redundant (does not make a difference). You will notice that nothing will change even if you remove this part.

The `bg-primary` variable automatically changes through [HERE](https://github.com/9inpachi/krit/blob/master/src/index.scss#L12-L17) so you don't need to add that.

And the `.menu-item` hover is handled by the `CircleButton` component [HERE](https://github.com/9inpachi/krit/blob/master/src/components/CircleButton/CircleButton.scss#L20) in the exact same way so that is also covered.

9inpachi replied:
You just need to add the `background-color` for `.menu-item` which you did on line 5.

Thanks for your contribution Usama!

Usamaahmad751 replied:
Yeah, I see. The styling in .dark-theme doesn't make any sense here. Only the thing we need to do is to add background-color for .menu-item.  I minimized the code and remove the redundant code. Now you can review the changes. Thanks, Fawad for quick review.

Usamaahmad751 committed:
Removing redundant code

Some styles were not making much sense in the file. So I removed this code to minimize the changes.

9inpachi commented:
Thank you so much for your contribution @Usamaahmad751 !
